src/main/java/Command.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Command class.
+ */
+public abstract class Command {
+    private String inputCommand; //input command by the user
+    private boolean isExit;
+
+    /**
+     * Constructor.
+     * @param inputCommand command inputted by user.
+     * @param isExit whether to exit the chat bot or not.
+     */
+    public Command(String inputCommand, boolean isExit) {
+        this.inputCommand = inputCommand;
+        this.isExit = isExit;
+    }
+
+    /**
+     * Return the command inputted by user.
+     * @return
+     */
+    public String getInputCommand() {
+        return this.inputCommand;
+    }
+
+    /**
+     * Execute the given task.
+     * @param ui the Ui class in charge of aesthetics.
+     * @param list tasklist that stores all the task.
+     * @param storage storage that writes to file.
+     * @throws DukeException exception.
```

Comment:
It's better to specify the condition that will result in the exception, instead of just stating that it is an exception.